example/ck_tile/01_fmha/bias.hpp

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2024, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <ostream>
+#include <string>
+#include "ck_tile/core.hpp"
+#include "ck_tile/ops/fmha.hpp"
+
+// keep sync with BlockAttentionBiasEnum
+enum class bias_enum
+{
+    no_bias          = 0,
+    elementwise_bias = 1,
+    alibi            = 2,
+};
+
+struct bias_info
+{
+    bias_enum type;
+    /*
+     * simple dispatch logic
+     *
+     * if type == elementwise_bias:
+     *      if extra_info == 0:
+     *           bias is 1*1*s*s
+     *      elif extra_info == 1:
+     *           bias is 1*h*s*s
+     *      elif extra_info == 2:
+     *           bias is b*h*s*s
+     *
+     * elif type == alibi:
+     *       if extra_info == 0:
+     *           alibi in 1*h
+     *       elif extra_info == 1:
+     *           alibi in b*h
+     */
+    int extra_info;
+
+    void serialize(std::ostream& os) const
+    {
+        if(type == bias_enum::no_bias)
+            os << "n";
+        else if(type == bias_enum::elementwise_bias)
+        {
+            os << "e";
+            if(extra_info != 0)
+            {
+                os << "[" << extra_info << "]";
+            }
+        }
+        else if(type == bias_enum::alibi)
+        {
+            os << "alibi";
+            if(extra_info != 0)
+            {
+                os << "[" << extra_info << "]";
+            }
+        }
+    }
+
+    static bias_info decode(std::string str)
```

Comment:
it's better to use `std::string_view` as parameter type (it's cheap on copy)

Reply:
indeed this `decode()` will have input coming from `arg_parser`, where internal using string to hold data and return string.